hil-test/README.md

```diff
@@ -61,19 +59,24 @@ Our Virtual Machines have the following setup:
 - ESP32-C3 (`rustboard`):
   - Devkit: `ESP32-C3-DevKit-RUST-1` connected via USB-Serial-JTAG.
     - `GPIO2` and `GPIO4` are connected.
-  - VM: Ubuntu 20.04.5 configured with the following [setup](#vm-setup)
+  - RPi: Raspbian 12 configured with the following [setup](#vm-setup)
 - ESP32-C6 (`esp32c6-usb`):
   - Devkit: `ESP32-C6-DevKitC-1 V1.2` connected via USB-Serial-JTAG (`USB` port).
     - `GPIO2` and `GPIO4` are connected.
   - RPi: Raspbian 12 configured with the following [setup](#vm-setup)
-- ESP32-H2 (`esp32h2-usb`):
+- ESP32-H2 (`esp32h2-usb`): Currently disabled
```

Comment:
Nitpick: This comment might be misleading, some tests are allowed on H2.

Reply:
Ups, good catch! I forgot to remove this as H2 was added later on